### PR TITLE
#184948280 Ft Track Order Status

### PR DESCRIPTION
**Pull request by ericndungutse**
#### What does this PR do?
This PR implements **track order status** feature

#### Description of Task to be completed?

- [x] Create orders page.
- [x] Create order detail page.
- [x]  Integrate with backend

#### How should this be manually tested?
- In your project terminal, run `git fetch origin pull/50/head:ft-track-order-status-184948280`
- Checkout to **ft-track-order-status-184948280** branch
- Install all the dependencies by running npm install
- create `.env` file and set `LIVE_API_URL` to server api `url`
- Then run the project by typing `npm run dev`

#### Routes to be tested:
- `customer/orders/history>`
- `customer/orders/<order-id>`

#### What are the relevant pivotal tracker/Trello stories?
Story ID: [184948280](https://www.pivotaltracker.com/story/show/184948280)

#### Screenshoots
Screenshot (28)

Screenshot (29)

### CODE REVIEW

**Review by khevin2 — COMMENTED** [review]
image
I am getting Page not found error

File: q [hunk 1]
```diff
@@ -0,0 +1,79 @@
+[33m6ef411f[m[33m ([m[1;36mHEAD -> [m[1;32mft-review-products-184948277[m[33m)[m feat(review): create review
```
Comment (khevin2) [inline q]: I think this file was pushed by accident

**Review by khevin2 — APPROVED** [review]

**Comment by Emile-SHUMBUSHO** [comment]
Reaching to your hosted version seems not working.
Screenshot 2023-05-26 at 09 33 12

**Review by wilbrord2 — APPROVED** [review]

**Review by GasanaJr — APPROVED** [review]

**Review by Mystica52 — APPROVED** [review]

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**